web-app/__tests__/components/BookingModal.js

```diff
@@ -42,17 +45,20 @@ describe('BookingModal', () => {
   it('closeModal calls closeModal prop', () => {
     const mockOnClick = jest.fn();
 
-    const wrapper = shallow(<BookingModal 
-      closeModal={mockOnClick}
-      showModal 
-      booking={ {} }
-      changeStart={() => {}} 
-      changeEnd={() => {}} 
-      changeHalfday={() => {}} 
-      requestHoliday={() => {}} 
-      cancelHoliday={() => {}} 
-      updateHoliday={() => {}}
-      changeWFH={() => {}}  />);
+    const wrapper = shallow(
+      <BookingModal
+        closeModal={mockOnClick}
+        showModal
+        booking={{}}
+        changeStart={() => {}}
+        changeEnd={() => {}}
+        changeHalfday={() => {}}
+        requestHoliday={() => {}}
+        cancelHoliday={() => {}}
+        updateHoliday={() => {}}
+        changeWFH={() => {}}
+      />,
+    );
 
     expect(mockOnClick.mock.calls.length).toEqual(0);
```

Comment:
Not sure this is really necessary.

Reply:
Actually I think this whole area could just use:
`expect(mockOnClick).toBeCalled();`

https://jestjs.io/docs/en/mock-functions#custom-matchers